cmd/prometheus/main.go

```diff
@@ -780,10 +780,11 @@ func main() {
 	)
 
 	var (
-		localStorage  = &readyStorage{stats: tsdb.NewDBStats()}
-		scraper       = &readyScrapeManager{}
-		remoteStorage = remote.NewStorage(logger.With("component", "remote"), prometheus.DefaultRegisterer, localStorage.StartTime, localStoragePath, time.Duration(cfg.RemoteFlushDeadline), scraper)
-		fanoutStorage = storage.NewFanout(logger, localStorage, remoteStorage)
+		localStorage            = &readyStorage{stats: tsdb.NewDBStats()}
+		scraper                 = &readyScrapeManager{}
+		enableTypeAndUnitLabels = cfg.scrape.EnableTypeAndUnitLabels
```

Comment:
nit: Why this variable if we use it only once?

Reply:
[e2bd4a0](https://github.com/prometheus/prometheus/pull/17033/commits/e2bd4a0c5f4f618f0f65e0a0d41dbb5cf24ef399)
I thought that you mentioned to create a variable. I was wrong. MB. Thanks